tagged.cabal

```diff
@@ -73,5 +73,7 @@ library
 
   if flag(transformers)
     build-depends: transformers        >= 0.2 && < 0.6
```

Comment:
There are now two `build-depends: transformers` lines (this is one of them), which feels a bit strange. At the very least, I'd expect them to have non-overlapping version bounds.

Reply:
We can have

```
If impl(ghc >= 7.10)
  build-depends: transformers >= 0.4.2.0 && <0.6
else
  build-depends: transformers >= 0.2 && <0.6
```

The point of current "orphan" `if impl(ghc >= 7.10) build-depends: transformers >= 0.4.2.0` is to ensure that you don't try to install older transformers (without `Data.Functor.Classes` module). "why would anyone install old `transformers`", you might ask. I don't better question than "I, when checking that lower bounds are correct". Otherwise the conditionals/CPP for `Data.Functor.Classes` would be trickier.

Reply:
I can write that rationale down *somewhere*, but not sure if replicating to each `.cabal` file where we want conditional `transformers-compat` are good place**s**

Reply:
OK. I think a brief comment (something to the effect of "Ensure `Data.Functor.Classes` is always available") would be nice.

Reply:
amended